interfaces/repo.go

```diff
@@ -656,3 +656,35 @@ func (r *Repository) RemoveSnap(snapInfo *snap.Info) error {
 
 	return nil
 }
+
+// DisconnectSnap disconnects all the connections to and from a given snap.
+//
+// The return value is a list of snap.Info's that were affected.
```

Comment:
The only reason to call this function is because we want the known (!) snap reconfigured.